docs/resources/annotation.md

```diff
@@ -0,0 +1,97 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+subcategory: "Generic"
+layout: "aci"
+page_title: "ACI: aci_annotation"
+sidebar_current: "docs-aci-resource-aci_annotation"
+description: |-
+  Manages ACI Annotation
+---
+
+# aci_annotation #
+
+Manages ACI Annotation
+
```

Comment:
Add disclaimer of concurrent use with annotations attribute in new gen resources

Reply:
added warning